src/installer/utils.py

```diff
@@ -190,6 +190,9 @@ def parse_entrypoints(text):
 
     # Borrowed from https://github.com/python/importlib_metadata/blob/v3.4.0/importlib_metadata/__init__.py#L90  # noqa
```

Comment:
Is this comment still up-to-date @pradyunsg? The linked target is entirely different from the code below.

Reply:
Yea, this is out of date. Let’s drop it.

@sdispater could you make this change in this PR?

Reply:
Actually, nvm. Let’s do that in a follow up.

Reply:
@pradyunsg I did it anyway :-)